configuration/pih/reports/sql/covid19encounterexport.sql

```diff
@@ -6,98 +6,1328 @@
+    '161033',
+    'fr'
+);
+UPDATE temp_encounter SET breast_feed = OBS_VALUE_CODED_LIST(
+	encounter_id,
+    'CIEL',
+    '5632',
+    'fr'
+);
+UPDATE temp_encounter SET vaccination_up_todate = OBS_VALUE_CODED_LIST(
+	encounter_id,
+    'CIEL',
+    '5585',
+    'fr'
+);
+
+-- home medication
+UPDATE temp_encounter 
+SET 
+    home_medications = OBS_VALUE_TEXT(encounter_id, 'CIEL', '162165');
+
+UPDATE temp_encounter 
+SET 
+    allergies = OBS_VALUE_TEXT(encounter_id, 'CIEL', '162141');
+
+UPDATE temp_encounter te
+        LEFT JOIN
+    obs o ON o.concept_id = CONCEPT_FROM_MAPPING('CIEL', '1730')
+        AND o.person_id = te.patient_id
+        AND o.encounter_id = te.encounter_id
+        AND o.voided = 0 
+SET 
+    te.symptom_start_date = o.value_datetime;
+
+-- symptoms
+UPDATE temp_encounter 
+SET 
+    symptoms = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '1727', 'fr');
+
+-- other symptoms
+UPDATE temp_encounter 
+SET 
+    other_symptoms = OBS_VALUE_TEXT(encounter_id, 'CIEL', '165996');
+
+UPDATE temp_encounter 
+SET 
+    comorbidities = OBS_VALUE_CODED_LIST(encounter_id, 'PIH', '12976', 'fr');
+
+-- comorbidities
+UPDATE temp_encounter 
+SET 
+    available_comorbidities = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '162747', 'fr');
+
+-- other comorbidities
+UPDATE temp_encounter te
+        LEFT JOIN
+    obs o ON o.person_id = te.patient_id
+        AND o.encounter_id = te.encounter_id
+        AND o.voided = 0
+        AND o.concept_id = CONCEPT_FROM_MAPPING('CIEL', '162747')
+        AND o.value_coded = CONCEPT_FROM_MAPPING('CIEL', '5622') 
+SET 
+    other_comorbidities = o.comments;
+
+-- mh
+UPDATE temp_encounter 
+SET 
+    mental_health = OBS_VALUE_TEXT(encounter_id, 'CIEL', '163044');
+
+-- smoker
+UPDATE temp_encounter 
+SET 
+    smoker = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '163731', 'fr');
+
+-- transfer
+UPDATE temp_encounter 
+SET 
+    transfer = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '160563', 'fr');
+
+UPDATE temp_encounter 
+SET 
+    transfer_facility = OBS_VALUE_TEXT(encounter_id, 'CIEL', '161550');
+
+-- covid case contact
+UPDATE temp_encounter 
+SET 
+    covid_case_contact = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '162633', 'fr');
+
+-- case condition
+UPDATE temp_encounter 
+SET 
+    case_condition = OBS_VALUE_CODED_LIST(encounter_id, 'CIEL', '159640', 'fr');
+
+-- vitals
+UPDATE temp_encounter 
+SET 
+    temp = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '5088');
+
+UPDATE temp_encounter 
+SET 
+    heart_rate = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '5087');
+
+UPDATE temp_encounter 
+SET 
+    respiratory_rate = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '5242');
+
+UPDATE temp_encounter 
+SET 
+    bp_systolic = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '5085');
+
+UPDATE temp_encounter 
+SET 
+    bp_diastolic = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '5086');
+
+UPDATE temp_encounter 
+SET 
+    SpO2 = OBS_VALUE_NUMERIC(encounter_id, 'CIEL', '5092');
+
+-- room air
+UPDATE temp_encounter 
```

Comment:
Is it more readable to have this statement on 7 lines of code?  Can't we just make all of these 1 line of code?